core/server/src/main/java/alluxio/worker/block/BlockWorkerClientServiceHandler.java

```diff
@@ -196,23 +199,40 @@ public String call() throws AlluxioException, IOException {
   }
 
   /**
-   * Used to request space for some block file.
+   * Requests space for a block.
    *
    * @param sessionId the id of the client requesting space
    * @param blockId the id of the block to add the space to, this must be a temporary block
    * @param requestBytes the amount of bytes to add to the block
    * @return true if the worker successfully allocates space for the block on block’s location,
    *         false if there is no enough space
+   * @throws AlluxioTException if an Alluxio error occurs
+   * @throws ThriftIOException if an I/O error occurs
    */
   @Override
-  public boolean requestSpace(long sessionId, long blockId, long requestBytes) {
-    try {
-      mWorker.requestSpace(sessionId, blockId, requestBytes);
-      return true;
-    } catch (Exception e) {
-      LOG.error("Failed to request {} bytes for block: {}", requestBytes, blockId, e);
-    }
-    return false;
+  public boolean requestSpace(final long sessionId, final long blockId, final long requestBytes)
+      throws AlluxioTException, ThriftIOException {
+    return RpcUtils.call(new RpcCallable<Boolean>() {
```

Comment:
a side question: by using `RpcUtils.call`, are we increasing the load of GC as each RPC call will result in an object construction/desctruction.